test/Microsoft.Azure.Devices.Edge.Test.Common/config/EdgeConfigBuilder.cs

```diff
@@ -52,9 +52,11 @@ public IModuleConfigBuilder AddModule(string name, string image)
             return builder;
         }
 
-        /* Will output edgeHub and edgeAgent configurations, then a full configuration at the end.
-           This is done to assure routes are set up for the most recent $edgeHub deployment before the test modules start sending messages (i.e. assure messages won't get dropped).
-           Another way to handle this is to define all possible routes at the very beginning of the test, but there is added complexity as module names are assigned dynamically. */
+        // Returns two configurations: one with just $edgeAgent and $edgeHub; the other with
+        // everything. This is done to ensure edgeHub's routes are ready before the test modules
+        // start sending messages, to avoid dropped messages. Another way to handle this is to
+        // define all possible routes at the beginning of the test run, but there is added
+        // complexity as module names are assigned dynamically.
```

Comment:
nit: stylecop might complain about no /* syntax

Reply:
StyleCop doesn't complain. Actually I thought that our team coding standard was the opposite (prefer '//' to C-style '/* */'), although I can't find that written anywhere right now...I guess I was just confused. Anyway, if you look for instances of C-style comments in our codebase, the vast majority of them were added by newer devs in the last 6 months. 😃